src/main/java/Entities/CollaborativeTask.java

```diff
@@ -0,0 +1,217 @@
+package Entities;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+public class CollaborativeTask extends Task implements Timeblockable{
+    private boolean recurring;
+    private String frequency;
+    private LocalDateTime startTime;
+    private LocalDateTime endTime;
```

Comment:
For startTime and endTime, I realized that for scheduling collaborative tasks, the user can choose multiple time blocks so instead of startTime and endTime, it would have to be: private ArrayList<ArrayList<LocalDateTime>> timeBlocks (I don't know why it doesn't show up when I type it but the type of the inner array list is LocalDateTime, so it's an array list of array lists of LocalDateTime). I would also need getter and setter methods for timeBlocks.

Reply:
Should be all fixed.